ipn/local.go

```diff
@@ -1360,7 +1362,7 @@ func (b *LocalBackend) enterState(newState State) {
 		for _, addr := range b.netMap.Addresses {
 			addrs = append(addrs, addr.IP.String())
 		}
-		systemd.Status("Connected; %s; %s", strings.Join(addrs, " "), b.activeLogin)
+		systemd.Status("Connected; %s; %s", strings.Join(addrs, " "), activeLogin)
```

Comment:
I think I'd put the email address first, before the IPs, which will get pretty noisy soon here with IPv6 being added.